apps/02RockPaperScissors.js

```diff
@@ -12,37 +12,37 @@ if (hand1 === hand2) {
 }
 
 if (hand1 === 'rock') {
-   if (hand2 === 'scissors') {
+   else (hand2 === 'scissors') {
```

Comment:
This line 15 is what is showing the error. You cannot have an "else" statement INSIDE of an if statement. this is acceptable:

javascript```
if(hand2 === 'scissors') {
//do something }
else {
//do something else
}
```